crates/hyperqueue/src/bin/hq.rs

```diff
@@ -526,6 +567,9 @@ async fn main() -> hyperqueue::Result<()> {
         SubCommand::AutoAlloc(opts) => command_autoalloc(&gsettings, opts).await,
         SubCommand::EventLog(opts) => command_event_log(opts),
         SubCommand::GenerateCompletion(opts) => generate_completion(opts),
+        SubCommand::Task(TaskOpts {
```

Comment:
Let's move this branch (and also the Task enum variant in the SubCommand enum) closer to the job branches/variants.